content/docs/dvclive/api-reference/next_step.md

```diff
@@ -25,7 +25,7 @@ selected in [`dvclive.init()`] and whether `DVC` is available or not.
 
 If `summary` is True, on each `dvclive.next_step()` call, DVCLive will generate
 a summary of the values previously logged with [`dvclive.log()`], and increase
-the `step` count.
+the [`step`](/doc/dvclive/api-reference/get_step) count.
```

Comment:
There are some other places where `step` is mentioned. Should these all link to doc/dvclive/api-reference/get_step? This is not a rhetorical question, I'm honestly not sure what is best.

Reply:
Yes, I think those should all link to `get_step`.